components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -929,6 +929,19 @@ document.observe('xwiki:dom:loaded', function() {
     // Create and attach the suggests
     // -----------------------------------------------------------------
     if (typeof(PhenoTips.widgets.Suggest) != "undefined") {
+
+
+      // close all suggest pickers when clicking outside the picker and dialogues invoked by the picker
+      document.observe('mousedown', function(event) {
+        if (!event.findElement('.suggestItems') && !event.findElement('.dialog-ontology-browser')) {
```

Comment:
Yeah, for small fixes xar:format screws up a diff by introducing way too many unwanted changes